core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -21,186 +22,17 @@
  * @since 4.0
  */
 public interface RpcManager {
```

Comment:
Actually looking again we do consolidate it for the JGroups case so it would mostly only help in additional collection allocations that we would do internally.